src/app/services/mail/mail.utils.ts

```diff
@@ -319,11 +319,11 @@ export const generateIssueReportedNotificationHtml = ({
   return dedent`
   <p>Dear Form admin,</p>
   <p>Respondents are facing issues on your form: ${formTitle}.<p>
```

Comment:
/notFromThisPRButShoulStillFix

⚠️ for markup safety, `formTitle` should be html-encoded (with [html entities](https://developer.mozilla.org/en-US/docs/Glossary/Entity))

We can assume `formResultUrl` and `appName` are markup-safe (so no need to change those), but since the form-title is provided by users, we need to encode it properly to ensure we generate valid markup.

Reply:
Urgh, I see we have other markup content where this was not done 😢 (e.g. in `generatePaymentConfirmationHtml()`), We should upgrade all of those.

Reply:
Ok created https://linear.app/ogp/issue/FRM-1048/ensure-user-input-variable-is-html-encoded to track, will work on it after this

Reply:
Thanks @mayying 🙏